usaspending_api/references/models.py

```diff
@@ -482,8 +482,8 @@ class Meta:
         unique_together = (('program_activity_code', 'budget_year', 'responsible_agency_id', 'allocation_transfer_agency_id', 'main_account_code'),)
 
 
-class CFDA(DataSourceTrackedModel):
-    program_number = models.TextField(null=False)
+class Cfda(DataSourceTrackedModel):
+    program_number = models.TextField(null=False, db_index=True)
```

Comment:
Thanks for swinging back for the index. I _think_ this will put an index on the field (which is good), but not necessary a unique index. IIRC, using `unique=True` will automatically create a unique index.